src/utils/workspace.ts

```diff
@@ -86,8 +72,8 @@ export function getContainingWorkspace(fsPath: string): vscode.WorkspaceFolder |
 }
 
 export async function findFilesByFileExtension(fsPath: string | undefined, fileExtension: string): Promise<vscode.Uri[]> {
-    // if there is a fsPath, then only check the folder for the file extension, otherwise use all currently opened workspaces
-    const relativeDirectory: vscode.RelativePattern | string = fsPath ? new vscode.RelativePattern(fsPath, `*.${fileExtension}`) : path.join('**', `*.${fileExtension}`);
+    // if there is a fsPath, then only check the folder for the file extension, otherwise recursively check all currently opened workspaces
+    const relativeDirectory: vscode.RelativePattern | string = fsPath ? new vscode.RelativePattern(fsPath, `*.${fileExtension}`) : `**/*.{${fileExtension}}`;
```

Comment:
> Regarding the recursive search, it doesn't seem to take very long (less than a second). I have a pretty big temp folder with some recursive structure.

I will continue to stress this any time we're talking performance. You can test _relative_ performance (aka is one option slower than the other) with your default setup, but you can't test absolute performance (is this option too slow). Get an old machine with bad specs and slow internet and you _might_ be able to make an argument, but ultimately we just have to follow best practices (especially with any operation involving the file system or internet), and make decisions based on telemetry